app/utils/bind-generator.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * This utility helps to bind a generator function in a similar way that `Function.prototype.bind` does.
+ *
+ * Example:
+ *
+ * const gen = function* (param1, param2) {
+ *  yeild something();
```

Comment:
Typo on yield. Not super important since it's just part of comments, but can't hurt to fix now.

Reply:
Yeah.. And not the only one :) It was a bit in rush, but I've fixed them all and added some more examples to `dynamic-query`